docs/import/create_score_asset.js

```diff
@@ -319,47 +322,86 @@ function createScoreAsset(
     allStatesProcessing = allStatesProcessing.merge(processing);
   }
 
+  return backUpAssetAndStartTask(allStatesProcessing);
+}
+
+/**
+ * Renames current score asset to backup location, deleting backup if necessary,
+ * and kicks off export task.
+ * @param {ee.FeatureCollection} featureCollection Collection to export
+ * @return {Promise<!ee.batch.ExportTask>} Promise that resolves with task if
+ *     rename dance was successful
+ */
+function backUpAssetAndStartTask(featureCollection) {
   const scoreAssetPath = getScoreAssetPath();
   const oldScoreAssetPath = getBackupScoreAssetPath();
   const task = ee.batch.Export.table.toAsset(
-      allStatesProcessing,
+      featureCollection,
       scoreAssetPath.substring(scoreAssetPath.lastIndexOf('/') + 1),
       scoreAssetPath);
-  return new Promise((resolve, reject) => {
-    ee.data.deleteAsset(oldScoreAssetPath, (_, err) => {
-      if (err) {
-        if (err === 'Asset not found.') {
-          console.log(
-              'Old ' + oldScoreAssetPath + ' not present, did not delete it');
+  return renameAssetAsPromise(scoreAssetPath, oldScoreAssetPath)
+      .catch((renameErr) => {
+        console.log(renameErr);
+        // These checks aren't perfect detection, but best we can do.
+        if (renameErr.includes('does not exist')) {
+          console.log('Old ' + scoreAssetPath + ' not found, did not move it');
+        } else if (renameErr.includes('Cannot overwrite asset')) {
+          // Delete and try again.
+          return new Promise(
+              (deleteResolve, deleteReject) =>
+                  ee.data.deleteAsset(oldScoreAssetPath, (_, deleteErr) => {
+                    if (deleteErr) {
+                      // Don't try to recover here.
+                      // Don't show deletion error to user, but log it
+                      // somewhere.
+                      console.error(deleteErr);
+                      const message =
+                          'Error moving old score asset: ' + renameErr;
+                      setStatus(message);
```

Comment:
Comment says don't show error but we're setting status here? Unclear how we'd ever reach this place right?

Reply:
Clarified comment. We reach this place if there's an error deleting: there's a test case for it.